### DISCUSSION TITLE
Generate Migrations command: Support updating existing tables

### DISCUSSION THREAD

**Discussion by edalzell** [statamic-rad-pack/runway#326] [Feature Requests]
My use case:

- existing resource/table
- added 30 fields to blueprint
- need to create new migration

Ended up pointing site at new db, then generating migration for the resource, then changing that migration to update the table.

**Comment by duncanmcclean** [comment]
Expanding on this... we could probably have Runway listen to the `BlueprintSaved` event and generate a migration based on the diff of changes in the blueprint.

We *might* not even need the command anymore if it's all driven by listeners.

**Comment by duncanmcclean** [comment]
I'm going to close this in favour of #519, which will remove the need for the existing `generate-migrations` command.
